### PR TITLE
fix: ping SpecRef in ordering-safe way 

### PR DESCRIPTION
**Pull request by saschanaz**

### CODE REVIEW

File: tests/spec/core/biblio-spec.js [hunk 1]
```diff
@@ -59,18 +59,17 @@ describe("W3C — Bibliographic References", () => {
   `;
 
   afterAll(flushIframes);
-  let isSpecRefAvailable = false;
   const bibRefsURL = new URL("https://specref.herokuapp.com/bibrefs");
+  const specRefPing = fetch(bibRefsURL, { method: "HEAD" });
```
Comment (marcoscaceres) [inline tests/spec/core/biblio-spec.js]: let's move this to `beforeAll`, and `await` it there...
```
  specRefOk = (await specRefPing).ok;
```

File: tests/spec/core/biblio-spec.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ describe("W3C — Bibliographic References", () => {
     let ref = doc.querySelector("#bib-TestRef1 + dd");
     expect(ref).toBeTruthy();
     // This prevents Jasmine from taking down the whole test suite if SpecRef is down.
-    if (!isSpecRefAvailable) {
+    if (!(await specRefPing).ok) {
```
Comment (marcoscaceres) [inline tests/spec/core/biblio-spec.js]: Ooops... here needs updating too.

File: tests/spec/core/biblio-spec.js [hunk 3]
```diff
@@ -85,7 +84,7 @@ describe("W3C — Bibliographic References", () => {
     let ref = doc.querySelector("#bib-TestRef1 + dd");
     expect(ref).toBeTruthy();
     // This prevents Jasmine from taking down the whole test suite if SpecRef is down.
-    if (!(await specRefPing).ok) {
+    if (!specRefOk.ok) {
```
Comment (marcoscaceres) [inline tests/spec/core/biblio-spec.js]: Sorry, this should just be `if(!specRefOk){  ... }`  as `specRefOk` is just a boolean and won't have a `.ok` member, right?

**Comment by saschanaz** [comment]
Sorry for intermediate messes 🤣

**Comment by marcoscaceres** [comment]
So, I guess this one closes https://github.com/w3c/respec/issues/828

**Pull request merged**